src/ProcessedStory.ts

```diff
@@ -42,6 +44,9 @@ export default class ProcessedStory {
         rawStory.parameters?.axe?.runOptions,
         rawStory,
       ),
+      context: normalizeContext(rawStory.parameters?.axe?.context, rawStory) as
+        | Context
+        | undefined,
```

Comment:
Note the `as`. I didn't feel like creating a Zod schema for all the permutations the context object can take, so left it just checking the general shape.